lil_aretomo/aretomo.py

```diff
@@ -70,9 +69,7 @@ def align_tilt_series(
         gpu_ids=gpu_ids,
     )
     aretomo_output = AreTomoOutput(tilt_series_file=tilt_series_file, reconstruction_file=reconstruction_file)
-    if aretomo_output.contains_alignment_results is True and skip_if_completed is True:
-        print(f'{basename} has already been aligned in {output_directory}, skipping...')
-    else:
+    if aretomo_output.contains_alignment_results is False and skip_if_completed is False:
         with open(output_directory / 'log.txt', mode='w') as log:
             subprocess.run(command, stdout=log, stderr=log)
         if aretomo_output.contains_alignment_results is False:
```

Comment:
I think this logic is incorrect, it will not run if there is existing output and `skip_if_completed=False`

Reply:
opps changed to or